x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/openai/OpenAiResponseHandler.java

```diff
@@ -19,9 +21,25 @@
 
 import java.io.IOException;
 
+import static org.elasticsearch.core.Strings.format;
 import static org.elasticsearch.xpack.inference.external.http.HttpUtils.checkForEmptyBody;
 
 public class OpenAiResponseHandler extends BaseResponseHandler {
+    /**
+     * Rate limit headers taken from https://platform.openai.com/docs/guides/rate-limits/rate-limits-in-headers
+     */
+    // The maximum number of requests that are permitted before exhausting the rate limit.
+    static final String REQUESTS_LIMIT = "x-ratelimit-limit-requests";
+    // The maximum number of tokens that are permitted before exhausting the rate limit.
+    static final String TOKENS_LIMIT = "x-ratelimit-limit-tokens";
+    // The remaining number of requests that are permitted before exhausting the rate limit.
+    static final String REMAINING_REQUESTS = "x-ratelimit-remaining-requests";
+    // The remaining number of tokens that are permitted before exhausting the rate limit.
+    static final String REMAINING_TOKENS = "x-ratelimit-remaining-tokens";
+    // The time until the rate limit (based on requests) resets to its initial state.
+    static final String REQUESTS_LIMIT_RESET_TIME = "x-ratelimit-reset-requests";
```

Comment:
Just to double check, we're intentionally not logging these right?

Reply:
No not used, I removed the constants